Tests/PSGetFindModule.Tests.ps1

```diff
@@ -38,7 +39,7 @@ function SuiteSetup {
         Rename-Item $script:moduleSourcesFilePath $script:moduleSourcesBackupFilePath -Force
     }
 
-    GetAndSet-PSGetTestGalleryDetails -SetPSGallery
+    #GetAndSet-PSGetTestGalleryDetails -SetPSGallery
```

Comment:
This was for testing purposes locally.  I need to uncomment it out.  Since the prerelease work is not in INT yet, I needed to write my prerelease tests to run against my local gallery.  This function was wiping out my Local gallery registration, so I took it out.
When everything is done, I'd like to put the gallery prerelease work in INT and then run these tests again against INT, to verify the tests are working as expected.